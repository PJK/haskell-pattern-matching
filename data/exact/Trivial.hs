module Trivial where

-- Exhaustive
i :: Int -> Int
i _ | otherwise = 1
