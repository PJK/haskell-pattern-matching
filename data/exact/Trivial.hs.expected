{
    "tag": "AnalysisSuccess",
    "contents": [
        [],
        [
            [
                "i",
                [
                    [
                        [
                            {
                                "tag": "VariablePattern",
                                "contents": "~a"
                            }
                        ],
                        [
                            {
                                "tag": "WildcardPattern",
                                "contents": []
                            }
                        ]
                    ]
                ]
            ]
        ]
    ]
}
